### PR TITLE
Support gesture margin for default Scroll

### PR DESCRIPTION
**Pull request by dshiroma**
- Applying `Scroll` margin property to all RPC calls to scroll.

Example usage now works as expected:

```
ad.ui(res='com.android.chrome:id/compositor_view_holder')
    .scroll(margin=100)  # <--- wouldn't have done anything before
    .down(percent=70, speed=700)
```

- - -
This change is [Reviewable](https://reviewable.io/reviews/google/snippet-uiautomator/51)

### CODE REVIEW

**Comment by ko1in1u** [comment]
Please help to build and update apk [here](https://github.com/google/snippet-uiautomator/blob/4e881dba008108d8a808d763ce62d2043b00201d/snippet_uiautomator/android/app/uiautomator.apk) as well, thank you.

**Comment by dshiroma** [comment]
Added .apk file.

File: snippet_uiautomator/android/app/uiautomator.apk
Comment (ko1in1u) [inline snippet_uiautomator/android/app/uiautomator.apk:1]: Hmm, weird... don’t know why the apk file size increased by 150%. I'll patch this PR and check how it goes on my work space.

Comment (dshiroma) [reply snippet_uiautomator/android/app/uiautomator.apk:1]: Not sure why it did that, but I uploaded the smaller one now.

**Review by ko1in1u — APPROVED** [review]

**Review by eric100lin — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/snippet-uiautomator/51#-:-O2XWzCP5CALvIFmqxBU:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @dshiroma and @ko1in1u)

___
*[`snippet_uiautomator/android/app/src/main/java/com/google/android/mobly/snippet/uiautomator/UiObject2Snippet.java` line 313 at r3](https://reviewable.io/reviews/google/snippet-uiautomator/51#-O2XWbJS0mkpJ9W9e3Fx:-O2XWbJS0mkpJ9W9e3Fy:bzc0jq5) ([raw file](https://github.com/google/snippet-uiautomator/blob/9248690dd6d1a8d25aa72db87ed3b3a2d43310da/snippet_uiautomator/android/app/src/main/java/com/google/android/mobly/snippet/uiautomator/UiObject2Snippet.java#L313)):*
> ```Java
>       throws SelectorException {
>     Direction direction = Direction.valueOf(directionStr);
>     UiObject2 uiObject2 = selector.toUiObject2();
> ```

nit: it would be nice to declare these variables as final



_Suggestion:_
```Java
    final Direction direction = Direction.valueOf(directionStr);

    final UiObject2 uiObject2 = selector.toUiObject2();
```

**Comment by dshiroma** [comment]
> :lgtm:
> 
> > Reviewed 2 of 2 files at r1, 1 of 1 files at r3, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/google/snippet-uiautomator/51#-:-O2XWzCP5CALvIFmqxBU:bnfp4nl)_ status: all files reviewed, 2 unresolved discussions (waiting on @dshiroma and @ko1in1u)
> 
> _[`snippet_uiautomator/android/app/src/main/java/com/google/android/mobly/snippet/uiautomator/UiObject2Snippet.java` line 313 at r3](https://reviewable.io/reviews/google/snippet-uiautomator/51#-O2XWbJS0mkpJ9W9e3Fx:-O2XWbJS0mkpJ9W9e3Fy:bzc0jq5) ([raw file](https://github.com/google/snippet-uiautomator/blob/9248690dd6d1a8d25aa72db87ed3b3a2d43310da/snippet_uiautomator/android/app/src/main/java/com/google/android/mobly/snippet/uiautomator/UiObject2Snippet.java#L313)):_
> 
> > ```java
> >       throws SelectorException {
> >     Direction direction = Direction.valueOf(directionStr);
> >     UiObject2 uiObject2 = selector.toUiObject2();
> > ```
> 
> nit: it would be nice to declare these variables as final
> 
> _Suggestion:_
> 
> ```java
>     final Direction direction = Direction.valueOf(directionStr);
> 
>     final UiObject2 uiObject2 = selector.toUiObject2();
> ```

Done.

**Pull request merged**